src/main/java/usecase_essence_use/data_calculator/CalculatorStat.java

```diff
@@ -1,17 +1,15 @@
-package usecase_heal_and_upgrade;
+package usecase_essence_use.data_calculator;
 
 public class CalculatorStat {
-    int statIncrease;
+    public CalculatorStat(){}
 
-    public CalculatorStat(){
-        this.statIncrease = 20;
-    }
+    public final int STAT_INCREASE = 20;
```

Comment:
naming convention incorrect again, only capitalize and underscore if it is static final

Reply:
fixed